apps/namadillo/src/atoms/integrations/functions.ts

```diff
@@ -224,7 +224,10 @@ export const getNamadaAssetByIbcAsset = (
       // Match native token(unam)
       counterpartyBaseDenom === namadaAsset.base ||
       // Match any other token
-      counterpartyBaseDenom === namadaAsset.traces?.[0].counterparty.base_denom
+      counterpartyBaseDenom ===
+        namadaAsset.traces?.[0].counterparty.base_denom ||
+      // For usdc
+      namadaAsset.traces?.[0].counterparty.base_denom === asset.base
```

Comment:
`console.log(counterpartyBaseDenom, asset.base, "counterpartyBaseDenom");`

`0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48 uusdc counterpartyBaseDenom`

Reply:
so counterparty base check doesn't work here but the the namadaAsset base denom does and matches the asset base....think this is *strictly* for USDC.